lib/tasks/gettext.rake

```diff
@@ -54,4 +55,11 @@ namespace :gettext do
       puts e.message
     end
   end
+
+  desc 'Merge all Pot files within locales folder'
+  task :merge_pot do
+    begin
+      GettextSetup::MergePot.merge_pot
+    end
```

Comment:
Like other rake tasks in this file, we may want to catch `GettextSetup::NoConfigFoundError`.